README.md

```diff
@@ -1 +1,28 @@
-# Mobile-Menu
\ No newline at end of file
+
+# Project Name:portfolio
+This is my first project at Microverse.
```

Comment:
- [ **OPTIONAL** ] Kindly make sure to improve your project description by describing what is about the project itself